### PR TITLE
Move the canisters tab to svelte

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
We would like to release the svelte version of the canisters tab.

# Changes
* Update REDIRECT_TO_LEGACY so that prod and staging use the svelte version of the canisters tab.
* Update documentation.
* Enable testnet population for "prod" builds; this requires svelte.

# Tests
- See CI.
- Also, I will test this with the redirect e2e test against a deployment to the nns-dapp testnet.....  Testing...done. OK

### CODE REVIEW

File: e2e-tests/specs/redirect-to-legacy.e2e.ts [hunk 1]
```diff
@@ -11,26 +11,26 @@ import { skipUnlessBrowserIs } from "../common/test";
 const REDIRECTS = {
```
Comment (bitdivine) [inline e2e-tests/specs/redirect-to-legacy.e2e.ts:11]: Note: The e2e tests are by default run on "both" only.  I would like to run this against "prod" on the testnet before we release, so I have updated the test.

File: deploy.sh [hunk 2]
```diff
@@ -220,8 +220,7 @@ if [[ "$POPULATE" == "true" ]]; then
   # Create users and neurons
   # Note: Cannot be used with flutter.
```
Comment (bitdivine) [inline deploy.sh:221]: This `--populate` command works only in svelte, which all tabs now are with a prod build.

File: frontend/dart/lib/data/env.dart [hunk 3]
```diff
@@ -2,4 +2,4 @@ const REDIRECT_TO_LEGACY = String.fromEnvironment('REDIRECT_TO_LEGACY');
 bool showAccountsRoute() { return ["flutter", "both"].contains(REDIRECT_TO_LEGACY); }
```
Comment (bitdivine) [inline frontend/dart/lib/data/env.dart:2]: Remove the canisters tab from flutter....

File: frontend/svelte/src/lib/constants/routes.constants.ts [hunk 4]
```diff
@@ -29,9 +29,12 @@ export const SHOW_PROPOSALS_ROUTE = [
   "prod",
```
Comment (bitdivine) [inline frontend/svelte/src/lib/constants/routes.constants.ts:29]: ... and add it to svelte.

**Review by peterpeterparker — APPROVED** [review]
Rubber stamp, thanks

**Pull request merged**